doc/lua_api.txt

```diff
@@ -3229,6 +3229,10 @@ These functions return the leftover itemstack.
     * `force_placement` is a boolean indicating whether nodes other than `air` and
       `ignore` are replaced by the schematic
     * Returns nil if the schematic could not be loaded.
+    * **Warning**: Once you have loaded a schematic from a file, it will be cached. Future calls
+      will always use the cached version and the replacement list defined for it,
+      regardless of weather the file or the replacement list parameter have changed.
```

Comment:
`regardless of whether the file or the replacement list parameter have changed.`

Reply:
Seems like already fixed.

Reply:
Thanks! Rubenwardy saw it as well and told me directly.